iron-doc-property-styles.html

```diff
@@ -94,13 +94,13 @@
       /* Metadata */
 
       #type {
-        @apply(--paper-font-code1);
+        @apply --paper-font-code1;
 
         font-style: italic;
       }
 
       .annotation {
-        color: var(--iron-doc-viewer-property-annotation-color, --paper-grey-700);
+        color: var(--iron-doc-viewer-property-annotation-color, var(--paper-grey-700));
```

Comment:
Conforms to "Update custom property syntax" @ https://www.polymer-project.org/2.0/docs/upgrade#shadow-dom-styles